### ISSUE TITLE
"conda install gemma" using bioconda couldn't work

### ISSUE THREAD

**Issue by Linghan-Jiang** [genetics-statistics/GEMMA#112]
I tried using bioconda to install gemma following "INSTALL.md", but error arised as follow:

```
    PackageNotFoundError: Packages missing in current channels:
            
       - gemma
     We have searched for the packages in the following channels:
            
      - https://conda.anaconda.org/bioconda/linux-64
      - https://conda.anaconda.org/bioconda/noarch
      - https://repo.continuum.io/pkgs/main/linux-64
      - https://repo.continuum.io/pkgs/main/noarch
      - https://repo.continuum.io/pkgs/free/linux-64
      - https://repo.continuum.io/pkgs/free/noarch
      - https://repo.continuum.io/pkgs/r/linux-64
      - https://repo.continuum.io/pkgs/r/noarch
      - https://repo.continuum.io/pkgs/pro/linux-64
      - https://repo.continuum.io/pkgs/pro/noarch
```

I googled the bioconda repository and searched for 'gemma' but found no record.

??? 

What should I do to fix this?

**Comment by pjotrp** [comment]
We still need to add it to conda.

**Comment by pjotrp** [comment]
Duplicate of https://github.com/genetics-statistics/GEMMA/issues/52

**Comment by qserenali** [comment]
Has this been fixed? I could only get 0.95alpha (released on 03/04/2016) to work (and is lagging several versions behind)


PackagesNotFoundError: The following packages are not available from current channels:

  - gemma

Current channels:

  - http://ddocent.com/bioconda/linux-64
  - http://ddocent.com/bioconda/noarch
  - https://conda.anaconda.org/BioConda/linux-64
  - https://conda.anaconda.org/BioConda/noarch
  - https://repo.continuum.io/pkgs/main/linux-64
  - https://repo.continuum.io/pkgs/main/noarch
  - https://repo.continuum.io/pkgs/free/linux-64
  - https://repo.continuum.io/pkgs/free/noarch
  - https://repo.continuum.io/pkgs/r/linux-64
  - https://repo.continuum.io/pkgs/r/noarch
  - https://repo.continuum.io/pkgs/pro/linux-64
  - https://repo.continuum.io/pkgs/pro/noarch

Is there another channel that I shall add? Searching https://anaconda.org/search?q=gemma found a moble/0.95alpha which seems to be related to this. I guess the newer versions have not been added? thanks!

**Comment by pjotrp** [comment]
See #52. If you would want to add gemma to bioconda - or get someone to do it, that would be great.
